app/test_split_integer.py

```diff
@@ -2,20 +2,35 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    value = 32
```

Comment:
The `value` is set to `32`, which is an integer, but the division operator `/` is used, which results in a float. If `split_integer` expects an integer, consider using `//` for integer division or directly assigning `32`.